### ISSUE TITLE
LP not connecting over Tesseracts 

### ISSUE THREAD

**Issue by Obbi89** [RS485/LogisticsPipes#945]
Hey,
since I find no other open Issue matching this. 
I'am on 0.9.3.99 (as of today up-to-date version) and I cannot get those tesseract connections to work.

I tried placing them in any order and on any side possible and they simply don't connect the two parts of a network. Maybe I got that whole support wrong.

But what I've tried is:
Basic + TE-Power Upgrade -> Tesseract 1
Tesseract 2 -> Basic + TE-Power Upgrade -> $someMachineBlock

On the second Tesseract the connection is red as in not connected to any other logistics pipes.
No Items can be requested from there. And items comming in from the other side just move straight through the network unrouted. (Which makes sense, since the network is unpowered and unconnected)

I thought connecting through tesseracts creates one network, that is as if it were directly attached to the other side.

Do I need to build two Networks and use the Inventory Connection Pipes? 
Cause that would make it the same as using ender chests in the first place.

Mods used:
- Forge
- Buildcraft. 
- all TeamCofH mods.
  All Mods are 1.7.10 and I use the latest available version as of today.

**Comment by bziemons** [comment]
You seem to do it right. Could you post a screenshot here on GitHub showing each of the tesseracts?

**Comment by Obbi89** [comment]
![2016-03-21_01 13 58](https://cloud.githubusercontent.com/assets/720039/13908415/63a21840-ef02-11e5-9e32-88963be0fe79.png)
I just quickly assembled that.

**Comment by SonOfTheStars** [comment]
are you sure boh tesseracts have all the stuffs set to "Send/Receive"? 
Because by default the are on Receive only

@theZorro266 Could it be that Tesseract Connections dont like to be power carriers?

**Comment by Obbi89** [comment]
Both are set on send and receive.

**Comment by SonOfTheStars** [comment]
so its on build 99...let me test some

**Comment by zortag** [comment]
i had this problem in build 72 too. reading other issues build 70 works as expected. i'l try that and i'll post again

**Comment by zortag** [comment]
no... tested with only lp build 100, bc7.1.14, bc compatibility 7.1.13, nei, thermal suite and dependencies. tested with java 7 and 8. no way to connect the pipes (the pipes connected to second tesseract stay red). the 2 tesseracts are setted up with all in input/output. creative world. in my case build 70 failed too. maybe the cause is buildcraft or thermal expansion?

**Comment by bziemons** [comment]
That is with LP 0.9.3.100 and newest BC:
![animation](https://cloud.githubusercontent.com/assets/1096713/13969628/5370a7ec-f084-11e5-81b1-c829f2ed2d08.gif)

I cannot reproduce this issue here. I haven't tried the newest Thermal _Whatever_ versions though.

**Comment by hron84** [comment]
I can watch this GIF for hours... :stuck_out_tongue:

**Comment by SonOfTheStars** [comment]
@theZorro266 have you tried having both ends have a Power-Transportation Upgrade and/or a TE-PowerSupplier Upgrade?

**Comment by zortag** [comment]
Thezorro266 can you list bc and cohf mods versions? I'm almost sure the solution is there.

**Comment by Obbi89** [comment]
@theZorro266 
I'm gonna update lp to 100 and check bc for updates.
Will report back then.

EDIT 1:
So no it's not working. 
I think I'll have a look into the building info and add some debug output to figure out where this is coming from.

EDIT 2:
Okay if I got this right, then you use an outdated api. As they moved around stuff in this commit:
https://github.com/CoFH/CoFHLib/commit/6c5c1db74c020ef1327cf1eff63cb5e86fad848b

It seems like they changed the EnderRegistry and the RegistryEnderAttuned class is gone and has been replaced by some other stuff. As far as I can see, your dummy "RegistryEnderAttuned" class kicks in and returns null so that stuff doesn't break.

It is as if there is no tesseract connected.
I will play around and see if changing this one class will fix stuff or if it takes more work than that.

EDIT 3:
Yes indeed it is.
![2016-03-25_09 04 18](https://cloud.githubusercontent.com/assets/720039/14040576/c48dd84c-f268-11e5-8d32-827fb70291c8.png)

Working now :+1: 

Here the Gist of what I quickly hacked down to test my theory. Might help you to fix/update that. Once you move to a newer version of TeamCofh stuff.

https://gist.github.com/Obbi89/e2f24f7d3a17a63a9651
